### ISSUE TITLE
Habilitar PowerBI

### ISSUE THREAD

**Issue by alanboy** [omegaup/omegaup#1595]
PowerBI: https://powerbi.microsoft.com/en-us/integrations/mysql/

el caso de uso es que el algunos miembros en la org puedan construir dashboards sin tener que cambiar codigo/consultas. la idea es que podamos responder preguntas mas rapido y experimentar con metricas.

una cosa es la implementacion y otra es la politica del uso interno de datos.

**Comment by lhchavez** [comment]
todavía no termino la política, pero me gustaría que los dashboards se originaran de consultas peer-reviewed y version-controlled. no se si powerbi soporte leer consultas de github.

**Comment by alanboy** [comment]
No, usa archivos (.pbix) que no son "texto plano" y por ende sera difícil de hacerles code review via github.

**Comment by lhchavez** [comment]
ok, busquemos alternativas entonces
